pkg/operator/ceph/csi/spec.go

```diff
@@ -380,11 +387,12 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 	pluginTolerations := getToleration(r.opConfig.Parameters, pluginTolerationsEnv, []corev1.Toleration{})
 	pluginNodeAffinity := getNodeAffinity(r.opConfig.Parameters, pluginNodeAffinityEnv, &corev1.NodeAffinity{})
 
-	if multusApplied && !v.SupportsMultus() {
-		return errors.Errorf("multus is applied but the csi version %q does not support it, need at least %q", v.String(), multusSupportedVersion.String())
+	if (!CSIParam.EnableCSIHostNetwork || multusApplied) && !v.SupportsNsenter() {
```

Comment:
How about another var that helps with readability? Otherwise the conditions are hard to understand.
```suggestion
	holderEnabled := !CSIParam.EnableCSIHostNetwork || multusApplied
	if (holderEnabled) && !v.SupportsNsenter() {
```